### ISSUE TITLE
w605 invalid exape sequence in comments

### ISSUE THREAD

**Issue by asottile** [PyCQA/flake8#771]
In GitLab by @GianniGi on Sep 20, 2020, 12:31

*Please describe how you installed Flake8*

$ pip install --user flake8

*Please describe the problem or feature*

I'm using flake8 with coc.nvim inside vim8.2

flake8 shows the w605 error for indentation problems inside comments. The comment can be made with ''' or with #, it shows the error in any casel.

*How to reproduce the problem*

save a file with extension .py and with this comment

```
print()



'''
classe \s
<r>[\s]</r>
# 1 condizione\.
# 1 condizione.{1,1}\.

'''
```

**Comment by asottile** [comment]
In GitLab by @GianniGi on Sep 20, 2020, 12:31

changed the description

**Comment by asottile** [comment]
In GitLab by @GianniGi on Sep 20, 2020, 12:32

changed the description

**Comment by asottile** [comment]
In GitLab by @asottile on Sep 20, 2020, 12:32

again, you didn't read or fill out the template

please stop and take the time to read -- it's right at the top:

```
<!--
*************************************************************************
NOTE: flake8 is a linting framework and does not implement any checks

if you are reporting a problem with a particular check, please track down
the plugin which implements that check.

some common ones:
- F###: https://github.com/pycqa/pyflakes
- E###, W###: https://github.com/pycqa/pycodestyle
*************************************************************************
-->
```

**Comment by asottile** [comment]
In GitLab by @asottile on Sep 20, 2020, 12:32

locked this issue
